### DISCUSSION TITLE
Aerodynamic jump

### DISCUSSION THREAD

**Discussion by gambon2010** [o-murphy/py-ballisticcalc#82] [Ideas]
If I am not mistaken the results of the current version of this library doesn't factor in the 'aerodynamic jump' (not sure if that's what its called in English). The calculations for this seem somewhat simple, I could work on adding it to this library if it is pertinent. 

@dbookstaber @o-murphy

**Comment by dbookstaber** [comment]
Yes!  I actually [have this as a feature note on my fork](https://github.com/dbookstaber/py_ballistics/issues/10) but haven't gotten to it, so it would be great if you can do it.

**Reply by gambon2010** [reply]
I'll look into implementing this in the next two weeks.

**Comment by o-murphy** [comment]
@dbookstaber @gambon2010 

There is the recommendations for new features adding process. We have to save backwards compatibility until major version still 2.x.x, so all new features must be implemented as optional and side by side with existing.

https://github.com/o-murphy/py-ballisticcalc/issues/32#issuecomment-2293737996

**Comment by o-murphy** [comment]
also please be correspond with new contribution terms

https://github.com/o-murphy/py-ballisticcalc/discussions/86

**Comment by o-murphy** [comment]
Should we make this functionality an optionally enabled in our engine? Or using this as always enables is good approach

**Reply by gambon2010** [reply]
For my personal application it would be useful to have an on/off toggle
